services/oada_upload/src/atomic-sleep.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'atomic-sleep';
```

Comment:
I think a proper type would be something like this. I haven't pulled, but does the code pass typechecks?

```suggestion
declare module "atomic-sleep" {
  function sleep(t: number): void;

  export default sleep;
}
```